src/theme/sidebar.css

```diff
@@ -0,0 +1,47 @@
+.sidebar {
+    width: 200px;
+    height: 100vh;
+    background-color: #333;
+    color: white;
+    position: fixed;
+    /* left: -200px; Hide sidebar by default */
+    transition: left 0.3s;
+  }
+  
+  .sidebar.open {
+    left: 0; /* Show sidebar when open */
+  }
+  
+  .burger {
+    position: absolute;
+    top: 20px;
+    left: 20px;
+    font-size: 24px;
+    cursor: pointer;
+    z-index: 1000;
+  }
+  
+  nav {
+    margin-top: 60px;
+  }
+  
+  ul {
+    list-style: none;
+    padding: 0;
+  }
+  
+  ul li {
+    padding: 15px 20px;
+    cursor: pointer;
+    display: flex;
+    justify-content: space-between;
+  }
+  
+  ul li:hover {
+    background-color: #444;
+  }
+  
+  .dropdown {
+    background-color: #444;
+    padding-left: 20px;
+  }
```

Comment:
My fault for not being clear, but we don't really want to be having css files. We can create side bars and dropdowns with mui and then apply css inline without needed a css file